src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -363,7 +365,7 @@ && new VersionNumber(coreCoordinates.version).compareTo(new VersionNumber("1.485
 
                     if(config.reportFile != null){
                         if(!config.reportFile.exists()){
-                            FileUtils.fileWrite(config.reportFile.getAbsolutePath(), "");
+                            FileUtils.touch(config.reportFile);
```

Comment:
Preferring Apache Commons FileUtils throughout this codebase to Plexus FileUtils.